src/main/java/com/sap/cloudfoundry/client/facade/domain/CloudServiceBinding.java

```diff
@@ -1,29 +1,22 @@
 package com.sap.cloudfoundry.client.facade.domain;
 
-import java.util.Map;
 import java.util.UUID;
 
+import org.cloudfoundry.Nullable;
 import org.immutables.value.Value;
 
 import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
 import com.fasterxml.jackson.databind.annotation.JsonSerialize;
-import com.sap.cloudfoundry.client.facade.Nullable;
 
 @Value.Immutable
 @JsonSerialize(as = ImmutableCloudServiceBinding.class)
 @JsonDeserialize(as = ImmutableCloudServiceBinding.class)
 public abstract class CloudServiceBinding extends CloudEntity implements Derivable<CloudServiceBinding> {
 
-    public abstract UUID getApplicationGuid();
-
-    @Nullable
-    public abstract Map<String, Object> getBindingOptions();
-
     @Nullable
-    public abstract Map<String, Object> getCredentials();
+    public abstract UUID getApplicationGuid();
 
-    @Nullable
-    public abstract String getSyslogDrainUrl();
```

Comment:
Why this property was deleted?

Reply:
The v3 service credential binding object (https://v3-apidocs.cloudfoundry.org/version/3.119.0/#the-service-credential-binding-object) does not have it